src/services/authenticator.js

```diff
@@ -27,9 +27,8 @@ function Authenticator() {
     }
   };
 
-  this.pathToForestrc = `${os.homedir()}/.forestrc`;
-
-  this.saveToken = (token) => fs.writeFileSync(this.pathToForestrc, token);
+  this.saveToken = (token, path = process.env.TOKEN_PATH || os.homedir()) => fs
+    .writeFileSync(`${path}/.forestrc`, token);
```

Comment:
Here I add the use of "TOKEN_PATH" variable. It allow tests that do not specify token mock (token:'any') to work with tokens in files. TOKEN_PATH is here to avoid overload our real token path.